### PR TITLE
#171203704 Update README

### PR DESCRIPTION
**Pull request by Niyonsengaeric**
#### What does this PR do?
 This PR ensures that the README file has all the information required to help a user to use the system.
#### Description of Task to be completed?
- Create a README file to help a user to interact with the system.

#### How should this be manually tested?
- Go to the [project](https://github.com/andela/knights-bn-backend/pull/21)

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#171203704](https://www.pivotaltracker.com/story/show/171203704)

#### Screenshots
N/A

### CODE REVIEW

**Review by Moise1 — APPROVED** [review]

**Comment by MuhanguziDavid** [comment]
Nice work with the README. However, the link (`Go to the project`) in your PR description should redirect to your branch, not the `develop` branch.

**Comment by MuhanguziDavid** [comment]
Ensure that your commit message follows agreed-upon guidelines
- Commit header
- No upper case letters in the header and body of the comment

File: README.md [hunk 1]
```diff
@@ -4,10 +4,64 @@
 [![Coverage Status](https://coveralls.io/repos/github/andela/knights-bn-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/knights-bn-backend?branch=develop)
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
 
-Barefoot Nomad - Making company travel and accomodation easy and convinient.
-=======
-
+# Barefoot Nomad
```
Comment (MuhanguziDavid) [inline README.md]: Let us have `Barefoot Nomad` as the first statement in the README instead of badges.

File: README.md [hunk 2]
```diff
@@ -4,10 +4,64 @@
...
+
+[Visual studio code](https://code.visualstudio.com/download) for editing and running the app
+
+[PostgreSQL](https://www.postgresql.org/download/) for Database
+
+### Installation
+
+##### A. Clone the project
+1. From your computer, open terminal 
+2. Run `https://github.com/andela/knights-bn-backend` OR [Download](https://github.com/andela/knights-bn-backend/archive/develop.zip) the project
```
Comment (MuhanguziDavid) [inline README.md]: Use the full command for cloning the repo. Also mention what the command does.

File: README.md (hunk 1 above)
Comment (Niyonsengaeric) [reply README.md]: you mean the title and the description ? or just the title only!

File: README.md [hunk 3]
```diff
@@ -4,10 +4,64 @@
...
+
+##### B. Setting up the environment
+1. Create a file and name it `.env` in root directory
+2. Find a file named `.env.example`
+3. Copy & paste all environmental variables from `.env.example` to `.env`
```
Comment (MuhanguziDavid) [inline README.md]: `Copy & paste` may mislead people.
Use something like, "Use .env.example" as a blueprint for your .env file."

File: README.md [hunk 4]
```diff
@@ -4,10 +4,64 @@
...
+1. Run `npm install` to install all dependencies.
+2. Run `CREATE DATABASE databasename` to create your database.
+3. Run `npm run createTables` to create your database tables.
+4. Run `npm test` to test automatically if the is functioning properly as expected
+5. Run `npm start` to run the app in development environment. 
+6. copy URL`localhost:4000/api/v1/docs` to the browser for the API Documentation.
+
+7. copy URL`localhost:4000/api/v1/` to postman to start testing the endpoints.
+### API endpoints
+`- POST /auth/signup - Create user account`
```
Comment (MuhanguziDavid) [inline README.md:59]: Add a link to the project swagger documentation or a postman collection

Comment (Niyonsengaeric) [reply README.md:59]: it is not yet deployed!

File: README.md (hunk 1 above)
Comment (MuhanguziDavid) [reply README.md]: Have the title as `Barefoot Nomad` and remove the word badges.

File: README.md (hunk 4 above)
Comment (MuhanguziDavid) [reply README.md:59]: Alright. We shall add it when the app gets deployed.

File: README.md (hunk 1 above)
Comment (Niyonsengaeric) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (MuhanguziDavid) [reply README.md]: The full command starts with `git clone ...`

Comment (Niyonsengaeric) [reply README.md]: done

**Review by MuhanguziDavid — APPROVED** [review]

**Pull request merged**